runtime/onert/backend/ruy/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+set(LIB_ONERT_BACKEND_RUY onert_backend_ruy)
+
+nnfw_find_package(Ruy REQUIRED)
+
+file(GLOB_RECURSE SOURCES "*.cc")
+
+add_library(${LIB_ONERT_BACKEND_RUY} SHARED ${SOURCES})
+
+target_link_libraries(${LIB_ONERT_BACKEND_RUY} PRIVATE nnfw_lib_ruy)
```

Comment:
What difference btw this and below ruy?

Reply:
@YongseopKim `nnfw_lib_ruy` means kernel implementations that use ruy library. It is located in `compute/ruy`. `ruy` means external `RUY` library itself.